brainsets/utils/misc_utils.py

```
@@ -0,0 +1,41 @@
+_functions = [
+    "calc_sampling_rate",
+]
+
+__all__ = _functions
```

Ruff (0.15.12)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)